static/app/utils/fields/index.ts

```diff
@@ -403,6 +403,45 @@ export enum IsFieldValues {
   UNLINKED = 'unlinked',
 }
 
+const IsFieldDescriptions = {
+  RESOLVED: t('Issues marked as fixed'),
+  UNRESOLVED: t('Issues still active and needing attention'),
+  ARCHIVED: t('Issues that have been archived'),
+  ESCALATING: t('Issues occurring significantly more often than they used to'),
+  NEW: t('Issues that first occurred in the last 7 days'),
+  ONGOING: t('Issues created more than 7 days ago or manually been marked as reviewed'),
+  REGRESSED: t('Issues resolved then occurred again'),
+  ASSIGNED: t('Issues assigned to a team member'),
+  UNASSIGNED: t('Issues not assigned to anyone'),
+  FOR_REVIEW: t('Issues pending review'),
+  LINKED: t('Issues linked to other issues'),
+  UNLINKED: t('Issues not linked to other issues'),
+};
+
+const FIELD_VALUE_TO_FIELD_DESCRIPTION = {
```

Comment:
nit: I would type this as `Record<IsFieldValue, string>` and just do

```ts
[IsFieldValue.RESOVLED]: t('Issues marked as fixed'),
// ...etc
```

That way if any new values get added, this type will error until a new string is added